src/service/RoomIdContext.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { RoomIDContextTypes } from "types";
+
+const RoomIDContext = React.createContext<RoomIDContextTypes>({ roomID: "", setRoomID: () => "" });
+
+const RoomIDProvider: React.FC<React.ReactNode> = ({ children }) => {
+  const [roomID, setRoomID] = React.useState("");
+
+  return <RoomIDContext.Provider value={{ roomID, setRoomID }}>{children}</RoomIDContext.Provider>;
+};
+
+const useRoomID = (): RoomIDContextTypes => {
+  const contextValue = React.useContext(RoomIDContext);
+  if (contextValue === undefined) {
+    throw new Error("useRoomId must be used within RoomIdProvider");
+  }
+
+  return contextValue;
+};
+
+export { RoomIDProvider, RoomIDContext, useRoomID };
```

Comment:
for consistency can you please follow how the exports are done in other context files
i.e.

``` ts
export const RoomIDContext = React.createContext<RoomIDContextTypes>({ roomID: "", setRoomID: () => "" });
export const useRoomID ...
export default RoomIDProvider
```

Reply:
sure, I did like that, because it is a convention in typescript

Reply:
I did the requested changes